src/materials/03-organisms/app-card-tabs.html

```diff
@@ -0,0 +1,266 @@
+<div class="app-card form-card tabs-card max-width">
+  <div class="app-inner header-wide">
+    <h2 class="app-card_h2">Current Contact Information</h2>
+  </div>
+
+  <div class="app-inner inset-wide border-bottom">
+    <div class="form-grid row expand">
+      <div class="form-grid_item small-12 medium-6 large-3 column">
+        <div class="form-group">
+          {{> forms.text-input }}
+        </div>
+      </div>
+      <div class="form-grid_item small-12 medium-6 large-3 column">
+        <div class="form-group">
+          {{> forms.text-input }}
+        </div>
+      </div>
+      <div class="form-grid_item small-12 medium-6 large-3 column">
+        <div class="form-group">
+          {{> forms.text-input }}
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="app-inner subheader-wide">
+    <h3 class="app-card_h3 t-sans">Current Mailing Address</h3>
+    <p class="form-note max-width">If the applicant reports a different address than at time of application, please pay extra attention to their original proof of address.</p>
+  </div>
+
+  <div class="app-inner inset-wide border-bottom">
+    <div class="form-grid row expand">
+      <div class="form-grid_item small-12 medium-6 large-3 column">
+        <div class="form-group">
+          {{> forms.text-input }}
+        </div>
+      </div>
+      <div class="form-grid_item small-12 medium-6 large-3 column">
+        <div class="form-group">
+          {{> forms.text-input }}
+        </div>
+      </div>
+    </div>
+    <div class="form-grid row expand">
+      <div class="form-grid_item small-12 medium-6 large-3 column">
+        <div class="form-group">
+          {{> forms.text-input }}
+        </div>
+      </div>
+      <div class="form-grid_item small-12 medium-6 large-3 column">
+        <div class="row">
+          <div class="small-6 column">
+            <div class="form-group">
+              <label for="" class="">{{default label "Text label"}}</label>
+              {{> forms.select }}
+            </div>
+          </div>
+          <div class="small-6 column">
+            <div class="form-group">
+              {{> forms.text-input }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="app-inner header-wide">
+    <h2 class="app-card_h2">Confirmed Preferences</h2>
+  </div>
+
+  <div class="app-inner inset-wide border-bottom">
+    <div class="form-grid row expand">
+      <div class="small-12 column">
```

Comment:
@slowbot how do you feel about adding the `.scrollable-table-container` class around the Confirmed Preferences table here, to demonstrate the scrollable ability? obviously we won't always want to scroll this table by default, as you said it depends on the content, but i think adding the scrollability here will make it easier to see that scrolling is one option, and also will prevent ppl from thinking something's gone wrong on the page